### DISCUSSION TITLE
ESP32 Wroom-32 -Board bootet nicht korrekt

### DISCUSSION THREAD

**Discussion by mpro1978** [frickelzeugs/FingerprintDoorbell#42] [Q&A]
Hallo zusammen, das Projekt ist genial! Meine Nachbaufähigkeiten leider etwas dürftig... Habe mir die benötigten Teile organisiert, und kriege die Sache aber nicht gebacken. Kurze Frage am Rande... Ich habe die Installationsanleitung nun x mal durchgespielt. Ich komme immer bis zum Ende vom Installationsbefehl: Leaving... Hard resetting via RTS pin...

Danach passiert aber leider nichts. Das heißt die rote (am Board) LED leuchtet dauerhaft und die blaue LED blinkt gelegentlich. Im Browser tut sich leider gar nichts (nach IP Eingabe Zeitüberschreitung). Am Fingerprintreader ist alles finster... 

Ist es möglich das das Board von AZ Delivery nicht geeignet ist? Oder noch etwas zusätzliches installiert werden muss?

![IMG_20221121_204955](https://user-images.githubusercontent.com/28113294/203147677-a6e8cda2-0553-4d81-a805-03c87900ddd1.jpg)

Ich hoffe jemand hat eine gute Idee.

Beste Grüße

Markus

**Comment by Dobs0001** [comment]
Das Board funktioniert zu 100%. Ich habe das gleiche. Hast du gesehen, das du am Fingerprint Reader 2x die 3,3Volt brauchst?

**Reply by baetzmr** [reply]
Installierst Du mit Visual Studio Code und PlattformIO? Firmware und Filesystem hochgeladen? Das mit dem gelegentlich blau leuchten hört sich komisch an... ich denk mal da fehlt noch was. Nach erfolgreichen Uploads sollte wenn alles sauber hochgeladen wurde erst mal der WiFi Config Mode starten - LED Breathing red.

**Reply by mpro1978** [reply]
> Das Board funktioniert zu 100%. Ich habe das gleiche. Hast du gesehen, das du am Fingerprint Reader 2x die 3,3Volt brauchst?

Vielen Dank für das Feedback! Es tut schon mal wirklich gut zu hören ein geeignetes Board zu haben. Das spornt mich jetzt natürlich noch mehr an das Teil auch zum Laufen zu bringen...

Beste Grüße

Markus

**Reply by mpro1978** [reply]
> Installierst Du mit Visual Studio Code und PlattformIO? Firmware und Filesystem hochgeladen? Das mit dem gelegentlich blau leuchten hört sich komisch an... ich denk mal da fehlt noch was. Nach erfolgreichen Uploads sollte wenn alles sauber hochgeladen wurde erst mal der WiFi Config Mode starten - LED Breathing red.

Hallo noch mal, habe die Installation nach der Method 1: use precompiled binaries Variante gemacht (Dachte da kann nix schief gehen).
Ich habe das ja wohl hoffentlich richtig verstanden... 5 Dateien runterladen; in Ordner kopieren; über cmd aus diesem Ordner den Befehl:

esptool.py --chip esp32 --baud 460800 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 40m --flash_size detect 0x1000 bootloader_dio_40m.bin 0x8000 partitions.bin 0xe000 boot_app0.bin 0x10000 firmware.bin 2686976 spiffs.bin

ausführen und ... yeah

und dann über IP zum Web Interface


Beste Grüße

Markus

**Reply by TheMaskedDeveloper** [reply]
> > Installierst Du 
> ausführen und ... yeah
> 
> und dann über IP zum Web Interface

Den Schritt, dass Du Dich mit dem WLAN das ESP32 verbindest (am Handy/Laptop/PC das WLAN suchen uns damit verbinden) hast Du auch gemacht?

**Reply by mpro1978** [reply]
Hallo zusammen,

habe jetzt die Installation auf einem neuen Board (selbes Modell) durchgeführt. 
Erfreulicherweise funktioniert jetzt alles wunderbar.

Ich bin sowas von begeistert!!! Vielen Dank an den Entwickler

Scheinbar hatte ich schlicht weg Pech...

Vielen Dank für die Unterstützung an alle.


Beste Grüße

Markus
